Content/Passives/CriticalPassives.cs

```diff
@@ -0,0 +1,57 @@
+﻿using PathOfTerraria.Core.Events;
+using PathOfTerraria.Core.Systems.TreeSystem;
+
+namespace PathOfTerraria.Content.Passives;
+
+/// <summary>
+/// Increases the chance of landing a critical strike by a flat amount.
+/// </summary>
+internal class AddedCriticalStrikeChance : Passive
+{
+	public override string InternalIdentifier => "AddedCriticalStrikeChance";
+	private float AmountPerLevel => 1.02f;
```

Comment:
Is there a reason this is a prop instead of a const?

Reply:
This persists for every new passive in this file.

Reply:
Updated